notebooks/405-paddleOCR-webcam/README.md

```diff
@@ -0,0 +1,32 @@
+# PaddleOCR with OpenVINO
+
+<p align="center" width="100%">
+    <img width="80%" src="https://raw.githubusercontent.com/yoyowz/classification/master/images/ezgif.com-gif-maker.gif"> 
+</p>
+
```

Comment:
Avoid deprecated HTML attributes and HTML itself. It's better to use markdown here like in 401.

Reply:
Changed to use markdown like in 401.